### PR TITLE
Update doc string links for accelerated Kafka datasource

### PR DESCRIPTION
**Pull request by chinmaychandak**
Updating doc string for #330.

### CODE REVIEW

File: streamz/sources.py [hunk 1]
```diff
@@ -597,27 +597,32 @@ def from_kafka_batched(topic, consumer_params, poll_interval='1s',
...
 
         Folks interested in trying out custreamz would benefit from this accelerated Kafka reader.
         If one does not want to use GPUs, they can use streamz as is, with the default engine=None.
 
-        To use this option, one must install RAPIDS custreamz using:
-        https://anaconda.org/rapidsai-nightly/custreamz
-        This will install all GPU dependencies, including streamz.
+        To use this option, one must install custreamz (use the appropriate CUDA version recipe) using
+        the following command, which will install all GPU dependencies, including streamz:
+        conda env create --name custreamz --file
```
Comment (martindurant) [inline streamz/sources.py:617]: Seems strange to require the user to create a new environment; also, this should only work with the raw version of the file, not the rendered github page

Comment (chinmaychandak) [reply streamz/sources.py:617]: > Seems strange to require the user to create a new environment

That's true, but the alternative is to install quite a few packages instead. The .yml script installs everything cleanly. Also, this is temporary (as mentioned at the end of the doc string) until the code gets merged into cudf/custreamz, after which it be a single line nightly package install. 

> this should only work with the raw version of the file, not the rendered github page

Correct, I was assuming people would understand this by themselves, that they would need to download the .yml file and then run the command pointing it to that file.

Comment (martindurant) [reply streamz/sources.py:617]: Perhaps better to write this out in full, even if it is only temporary

Comment (chinmaychandak) [reply streamz/sources.py]: Done

**Pull request merged**